lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -77,6 +77,24 @@ sub register ($self, $app, $config) {
             return $c->include_branding('external_reporting', %args);
         });
 
+    $app->helper(
+        format_file_size => sub {
+            my ($c, $size) = @_;
+            my $exp = 0;
+
+
+            my @units = qw(B KB MB GB TB PB);
+
+            for (@units) {
+                last if $size < 1024;
+                $size /= 1024;
+                $exp++;
+            }
+
```

Comment:
Please use OpenQA::Utils::human_readable_size

Reply:
Sure, thx for a review.